### PR TITLE
Update actions for Issue Alerts

### PR DESCRIPTION
**Pull request by MimiDumpling**
Sending an email notification is now separate from alerts. This PR reflects that update.

**BEFORE**:
image

**AFTER:**
image

### CODE REVIEW

**Comment by wedamija** [comment]
@MimiDumpling All the previous actions are still in place. The key changes here are:
 - `Send a notification to a single legacy service` has changed. It won't have the option for `Mail`, but if other legacy plugins are enabled they will be available there. If there are no legacy plugins enabled, then this option is completely hidden.
 - We added the `Send an Email` option. It's probably worth discussing what the sub-options are here, so that people know what `Issue Owners`, `Member`, and `Team` do specifically.

The global slack and pagerduty integrations still exist as well. Let me know if you'd like a screenshot of the sentry setup, since it has both slack and pagerduty set up.

**Comment by MimiDumpling** [comment]
> @MimiDumpling All the previous actions are still in place. The key changes here are:
> 
> * `Send a notification to a single legacy service` has changed. It won't have the option for `Mail`, but if other legacy plugins are enabled they will be available there. If there are no legacy plugins enabled, then this option is completely hidden.
> * We added the `Send an Email` option. It's probably worth discussing what the sub-options are here, so that people know what `Issue Owners`, `Member`, and `Team` do specifically.
> 
> The global slack and pagerduty integrations still exist as well. Let me know if you'd like a screenshot of the sentry setup, since it has both slack and pagerduty set up.

Thanks for taking a look, Dan. Will you Slack me a screenshot for the dropdown and any other relevant screenshots? 😊 Thank you!

**Comment by MimiDumpling** [comment]
@adhiraj @wedamija I updated the wording and screenshot. Let me know if there are other changes you'd like. 😊

File: src/collections/_documentation/workflow/alerts-notifications/alerts.md [hunk 1]
```diff
@@ -97,14 +97,17 @@ In the example, the alert will not fire because an event cannot satisfy both of
 
 The following actions are available:
 
+- Send an email
+    - Send email to either [Issue Owners]({%- link _documentation/workflow/issue-owners.md -%}), Team, or [Member]({%- link _documentation/accounts/membership.md -%}#member).
 - Send a notification to all legacy integrations
-    - [Legacy integrations]({%- link _documentation/workflow/integrations/legacy-integrations.md -%}), also known as Plugins, are configured per project. Legacy integrations currently include email.
-- Send a notification to a single legacy integration (service)
+    - [Legacy integrations]({%- link _documentation/workflow/integrations/legacy-integrations.md -%}), also known as Plugins, are configured per project. 
+- Send a notification to one legacy integration (service)
```
Comment (manuzope) [inline src/collections/_documentation/workflow/alerts-notifications/alerts.md:104]:
```suggestion
- Send a notification via an integration
```
This includes the following:
* Legacy integrations
* Custom integrations built using the Integration platform

File: src/collections/_documentation/workflow/alerts-notifications/alerts.md [hunk 2]
```diff
@@ -97,14 +97,17 @@ In the example, the alert will not fire because an event cannot satisfy both of
...
     - "service" is the same as legacy integration
```
Comment (manuzope) [inline src/collections/_documentation/workflow/alerts-notifications/alerts.md:105]: I'd remove this line cause this list contains more than just legacy integrations.

File: src/collections/_documentation/workflow/alerts-notifications/alerts.md [hunk 3]
```diff
@@ -97,14 +97,17 @@ In the example, the alert will not fire because an event cannot satisfy both of
...
+    - If no legacy integrations are enabled, this option is hidden.
```
Comment (manuzope) [inline src/collections/_documentation/workflow/alerts-notifications/alerts.md:106]:
```suggestion
    - If no legacy integrations or integrations built using the integration platform are enabled, this option is hidden.
```
You can wordsmith it but that's the general gist.

The unfortunately part of this grouping is that this also includes published apps. And our users don't know which published apps are built on our platform vs built by us. More of a heads up for @adhiraj.

File: src/collections/_documentation/workflow/alerts-notifications/alerts.md [hunk 4]
```diff
@@ -97,14 +97,19 @@ In the example, the alert will not fire because an event cannot satisfy both of
...
+- Send a notification via an integration
+    - This includes:
+        - Legacy integrations
+        - Integrations built using the [Integration platform]({%- link _documentation/workflow/integrations/integration-platform/index.md -%}) (includes published integrations, internal integrations, unpublished integrations)
+    -  If no legacy integrations or integrations built using the integration platform are enabled, this option is hidden.
 - Send a notification to a global integration, which includes
     - [PagerDuty]({%- link _documentation/workflow/integrations/global-integrations.md -%}#pagerduty)
     - [Slack]({%- link _documentation/workflow/integrations/global-integrations.md -%}#slack)
-    - Custom integrations built using the [Integration Platform]({%- link _documentation/workflow/integrations/integration-platform/index.md -%})
+    - Custom integrations built using the [Integration Platform]({%- link _documentation/workflow/integrations/integration-platform/index.md -%})
```
Comment (manuzope) [inline src/collections/_documentation/workflow/alerts-notifications/alerts.md:112]: Shoot sorry should have mentioned this in the last review. Custom integrations won't show up here.

Comment (MimiDumpling) [reply src/collections/_documentation/workflow/alerts-notifications/alerts.md]: no worries 😸

**Pull request merged**